includes/Extension/SearchReplace/views/replace-form.php

```diff
@@ -52,4 +52,7 @@
 			</button>
 		</p>
 	</form>
+	<?php if (!self::is_plugin_installed( $instawp_plugin )): ?>
+		<p class="notice notice-error update-nag">It is always a good idea to create a staging site before making changes <span id="notice-btn-toggle-replace-controls" class="button-link">Install Staging Plugin</span></p>
```

Comment:
An inline `span` element is treated as a button, which should instead be an actual button.

Reply:
Reolved in latest commit.